packages/react-core/src/components/DataList/DataListText.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/DataList/data-list';
+import { Tooltip } from '../Tooltip';
+import { DataListWrapModifier } from './DataList';
+
+export enum DataListTextVariant {
+  div = 'div',
+  nav = 'nav'
+}
+
+export interface DataListTextProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered within the data list text */
+  children?: React.ReactNode;
+  /** Additional classes added to the data list text */
+  className?: string;
+  /** Determines which element to render as a data list text */
+  variant?: DataListTextVariant | 'span' | 'div';
```

Comment:
Just want to double check this shouldn't be `component`? Typically when I've seen a prop to use a different element for something, it's a `component` prop.

Reply:
Fixed.